edx_proctoring/static/proctoring/js/views/proctored_exam_allowance_view.js

```diff
@@ -0,0 +1,144 @@
+var edx = edx || {};
+
+(function (Backbone, $, _) {
+    'use strict';
+
+    edx.instructor_dashboard = edx.instructor_dashboard || {};
+    edx.instructor_dashboard.proctoring = edx.instructor_dashboard.proctoring || {};
+
+    edx.instructor_dashboard.proctoring.ProctoredExamAllowanceView = Backbone.View.extend({
+        initialize: function (options) {
+            this.collection = new edx.instructor_dashboard.proctoring.ProctoredExamAllowanceCollection();
+            this.proctoredExamCollection = new edx.instructor_dashboard.proctoring.ProctoredExamCollection();
+            /* unfortunately we have to make some assumptions about what is being set up in HTML */
+            this.setElement($('.special-allowance-container'));
+            this.course_id = this.$el.data('course-id');
+
+            /* this should be moved to a 'data' attribute in HTML */
```

Comment:
Is this work that has been done?  Is being done?  Will be done?  If it's future work, please document as tech debt (a bug) in JIRA and record the issue number in the comment.